docs/samples/Microsoft.ML.Samples/Dynamic/GeneralizedAdditiveModels.cs

```diff
@@ -78,9 +78,9 @@ public static void RunExample()
             var studentTeacherRatioIndex = featureNames.ToList().FindIndex(str => str.Equals("TeacherRatio"));
 
             // Next, let's get the array of bin upper bounds from the model for this feature
```

Comment:
>bin upper bounds from the  [](start = 44, length = 26)

Please clarify meaning of word "bin" here.  Unclear if this is short for Binary or histogram bins or something else

Reply:
Done.